cinema/views.py

```diff
@@ -34,6 +38,29 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
```

Comment:
call super method instead of accesing this attribute directly
solve N+1 problem

Reply:
still not fixed